### PR TITLE
[KernelGen][Nvidia] Move hardswish operator to ops

### PR DESCRIPTION
**Pull request by YufanPeter**
### Summary
Move `hardswish` operator (`hardswish` and `hardswish.out`) from `experimental_ops` to `ops`

### Testing
- Parametrized correctness tests for `hardswish` and `hardswish.out` across multiple shapes and all supported float dtypes **(57 Passed)**
- Covered special values (±inf / ±0 / NaN and the piecewise knots at ±3 / ±6)
- Validated against the PyTorch reference with `--ref cpu`
- Tested on: NVIDIA H20

### Performance
Tested on NVIDIA H20 using:
```bash
pytest -s benchmark/test_hardswish.py --level core
```

#### hardswish

**float16**
| shape | Torch latency | Gems latency | speedup |
|---|---|---|---|
| [1073741824] | 1.267 | 1.154 | 1.098x |
| [64, 64] | 0.005 | 0.005 | 1.018x |
| [4096, 4096] | 0.024 | 0.023 | 1.030x |
| [64, 512, 512] | 0.024 | 0.023 | 1.026x |
| [1024, 1024, 1024] | 1.301 | 1.190 | 1.094x |

**float32**
| shape | Torch latency | Gems latency | speedup |
|---|---|---|---|
| [1073741824] | 2.077 | 2.024 | 1.026x |
| [64, 64] | 0.005 | 0.005 | 1.024x |
| [4096, 4096] | 0.037 | 0.038 | 0.974x |
| [64, 512, 512] | 0.037 | 0.039 | 0.950x |
| [1024, 1024, 1024] | 2.088 | 2.091 | 0.999x |

**bfloat16**
| shape | Torch latency | Gems latency | speedup |
|---|---|---|---|
| [1073741824] | 1.238 | 1.158 | 1.069x |
| [64, 64] | 0.006 | 0.005 | 1.079x |
| [4096, 4096] | 0.024 | 0.023 | 1.027x |
| [64, 512, 512] | 0.024 | 0.024 | 1.022x |
| [1024, 1024, 1024] | 1.333 | 1.182 | 1.128x |

#### hardswish_out

**float16**
| shape | Torch latency | Gems latency | speedup |
|---|---|---|---|
| [1073741824] | 1.259 | 1.159 | 1.087x |
| [64, 64] | 0.005 | 0.006 | 0.971x |
| [4096, 4096] | 0.024 | 0.023 | 1.032x |
| [64, 512, 512] | 0.024 | 0.024 | 1.019x |
| [1024, 1024, 1024] | 1.351 | 1.214 | 1.113x |

**float32**
| shape | Torch latency | Gems latency | speedup |
|---|---|---|---|
| [1073741824] | 2.049 | 2.079 | 0.986x |
| [64, 64] | 0.006 | 0.006 | 1.017x |
| [4096, 4096] | 0.037 | 0.039 | 0.962x |
| [64, 512, 512] | 0.038 | 0.040 | 0.944x |
| [1024, 1024, 1024] | 2.130 | 2.082 | 1.023x |

**bfloat16**
| shape | Torch latency | Gems latency | speedup |
|---|---|---|---|
| [1073741824] | 1.238 | 1.198 | 1.033x |
| [64, 64] | 0.005 | 0.005 | 1.057x |
| [4096, 4096] | 0.024 | 0.023 | 1.029x |
| [64, 512, 512] | 0.024 | 0.023 | 1.032x |
| [1024, 1024, 1024] | 1.251 | 1.199 | 1.044x |

#### Average
Combined average speedup: **1.030x**

### Files changed
- `src/flag_gems/ops/hardswish.py`: Production Triton kernel for `hardswish` and `hardswish.out`
- `tests/test_hardswish.py`: Accuracy tests for `hardswish` / `hardswish.out` (functional, out, special values)
- `benchmark/test_hardswish.py`: Performance benchmark using the standard benchmark framework
- `src/flag_gems/ops/__init__.py`: Export `hardswish` / `hardswish_out` through imports and `__all__`
- `src/flag_gems/__init__.py`: Register `hardswish` and `hardswish.out` in the aten patch list
- `conf/operators.yaml`: Add `hardswish` / `hardswish.out` entries `(kind: NeuralNetwork, stage: alpha 5.4)`
- `src/flag_gems/experimental_ops/hardswish.py`: Removed after migration to `ops`
- `experimental_tests/unit/hardswish_test.py`: Removed after migration to `ops`
- `experimental_tests/performance/hardswish_test.py`: Removed after migration to `ops`

### CODE REVIEW

File: benchmark/test_hardswish.py [hunk 1]
```diff
@@ -32,3 +32,23 @@ def test_hardswish_inplace():
...
+def test_hardswish():
+    bench = base.UnaryPointwiseBenchmark(
+        op_name="hardswish",
+        torch_op=torch.nn.functional.hardswish,
+        dtypes=consts.FLOAT_DTYPES,
+    )
+    bench.run()
+
+
+@pytest.mark.hardswish
```
Comment (bin913) [inline benchmark/test_hardswish.py]: Please keep the names consistent

Comment (YufanPeter) [reply benchmark/test_hardswish.py]: Should be **/test|hardswish:h20** instead of /test|hardwish:h20. **Might missing an 's'**?

Comment (bin913) [inline benchmark/test_hardswish.py]:
```suggestion
@pytest.mark.hardswish_out
```

**Review by bin913 — APPROVED** [review]
